### PR TITLE
CNV#25663: kernel samepage merging

### PR DESCRIPTION
**Pull request by ousleyp**
Version(s): 4.15+

Issue: [CNV-25663](https://issues.redhat.com//browse/CNV-25663) and [CNV-33130](https://issues.redhat.com//browse/CNV-33130)

Link to docs preview: https://71325--ocpdocs-pr.netlify.app/openshift-enterprise/latest/virt/virtual_machines/advanced_vm_management/virt-activating-ksm

QE review:
- [x] QE has approved this change.

### CODE REVIEW

File: modules/virt-configure-ksm-cli.adoc [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+  name: kubevirt-hyperconverged
+  namespace: {CNVNamespace}
+spec:
+  configuration:
+    ksmConfiguration:
+      nodeLabelSelector: {}
+# ...
+----
+
+* To enable the KSM activation feature on a subset of nodes, edit the `nodeLabelSelector` field. Add syntax that matches the nodes where you want {VirtProductName} to enable KSM. For example, this configuration allows {VirtProductName} to enable KSM on nodes where both `<first_example_key>` and `<second_example_key>` are set to `"true"`:
```
Comment (bburt-rh) [inline modules/virt-configure-ksm-cli.adoc:38]:
```suggestion
* To enable the KSM activation feature on a subset of nodes, edit the `nodeLabelSelector` field. Add syntax that matches the nodes where you want {VirtProductName} to enable KSM. For example, the following configuration allows {VirtProductName} to enable KSM on nodes where both `<first_example_key>` and `<second_example_key>` are set to `"true"`:
```
- Optional suggestion.

File: modules/virt-about-ksm.adoc [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+:_mod-docs-content-type: CONCEPT
+[id="virt-about-ksm_{context}"]
+= About using {VirtProductName} to activate KSM
+
+You can configure {VirtProductName} to activate KSM when nodes experience memory overload.
+
+[id="virt-ksm-configuration-methods"]
+== Configuration methods
+
+* To enable or disable this feature for all nodes, you can use the {product-title} web console or edit the `spec.configuration.ksmConfiguration` stanza of the `HyperConverged` custom resource (CR). Deleting the `ksmConfiguration` stanza from the `HyperConverged` CR disables the feature.
```
Comment (bburt-rh) [inline modules/virt-about-ksm.adoc:15]:
```suggestion
* To enable or disable this feature for all nodes, you can use the {product-title} web console or edit the `spec.configuration.ksmConfiguration` stanza of the `HyperConverged` custom resource (CR). Enable the feature by configuring settings in the `ksmConfiguration` stanza. Disable the feature by deleting the stanza from the `HyperConverged` CR.
```
- I was just wondering why there's a specific sentence about disabling the feature, but not a specific sentence about enabling it first. The paragraph begins with `To enable or disable this feature`, and that set my expectations that there would be info about enabling as well as disabling. It's not a big deal if you leave it as-is, but that was a question that came up for me when I read the paragraph, so maybe it's something for you to consider adding.

**Review by bburt-rh — APPROVED** [review]
Thanks. Excellent PR. Just a couple of minor comments/edits for you to consider. Otherwise, LGTM.

File: modules/virt-about-ksm.adoc (hunk 2 above)
Comment (ousleyp) [reply modules/virt-about-ksm.adoc]: Thank you! I decided to rework this section to make it (hopefully) less confusing and more minimalistic.

**Pull request merged**